packages/playkit-js-scripts/scripts/contrib.js

```diff
@@ -0,0 +1,75 @@
+const commander = require('commander');
+const path = require('path');
+const spawn = require('cross-spawn');
+const packageJson = require('../package.json');
+const VARIABLES = require('../config/variables.config');
+const chalk = require('chalk');
+
+const contribTypes = {
+    LATEST: 'latest',
+    NEXT: 'next',
+    LOCAL: 'local',
+};
+
+const program = new commander.Command(packageJson.name)
+    .version(packageJson.version)
+    .option(
+        '--type {latest|next|local}',
+        'choose a type of the contrib script'
+    )
+    .parse(process.argv);
+
+(async () => {
+    const contribTypesValues = Object.keys(contribTypes).map(key => contribTypes[key]);
+
+    if (!program.type || !contribTypesValues.includes(program.type)) {
+        return console.error(`
+        ${chalk.red('Please, provide the correct type parameter for the contrib script.')}
+            Types could be:
+                - ${contribTypes.LATEST}
+                - ${contribTypes.NEXT}
+                - ${contribTypes.LOCAL}
+                
+            For example:> kcontrib contrib --type=local
+    `);
+    }
+
+    const appPath = process.cwd();
```

Comment:
this is new requirement, I will update jira as well:
1. show the following message to the user: `The following packages will be re-installed from npm with tag {latest|next}`
2. for local type message is 
```
The following packages will be linked to local libraries. 
if error happens you should run `npm run setup` in the contrib repository
```

![image](https://user-images.githubusercontent.com/4751797/70248756-1691d780-1784-11ea-9bec-aea2d42ed02a.png)